README.md

```diff
@@ -452,6 +453,40 @@ bash ./scripts/build-and-test.sh
 
 **Note:** *You may need to update the emulator device to match your Xcode version*
 
+## Modify network requests
+
+You can track and modify network requests throughout OktaOidc by passing object that conforms to `OktaNetworkRequestCustomizationDelegate` protocol to the `requestCustomizationDelegate` property of `OktaOidcConfig` instance.
```

Comment:
@stevelind-okta , please help to validate spelling. Thank you! ;)

Reply:
Suggested change:
 
You can track and modify network requests made by `OktaOidc`. In order to do this, create an object conforming to the `OktaNetworkRequestCustomizationDelegate` protocol and set it to the `requestCustomizationDelegate` property on an `OktaOidcConfig` instance.

Reply:
Thank you, Steve! I'll update the readme.